src/components/radar/RadarAnimation.js

```diff
@@ -76,8 +78,22 @@ class RadarAnimation extends React.Component<Props> {
         position: MIDDLE_RIGHT,
     };
 
+    tetherRef = React.createRef<{ position: () => {} }>();
```

Comment:
I think this should be `React.createRef<TetherComponent>`

Reply:
best case it would be `React.createRef<typeof TetherComponent>`, but TetherComponent isnt typed so this is better

Reply:
If it became typed I think this would cause an error wouldn't it?

Reply:
If what became typed?

Reply:
I meant `TetherComponent`. I was thinking if the flow types got added for it, it would break. Anyways this is fine since the typing doesn't do much anyways